src/sentry/tasks/post_process.py

```diff
@@ -483,6 +485,10 @@ def should_update_escalating_metrics(event: Event) -> bool:
     time_limit=120,
     soft_time_limit=110,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=ingest_errors_tasks,
+        processing_deadline_duration=120,
+    ),
```

Comment:
This task and `save_event` are very high volume, we could introduce additional latency & load on memcache/postgres with all the additional option checks these will make. We should probably hold off on changing all of the ingest tasks until we have more confidence about the potential load impact in saas.

Reply:
Should we have a hardcoded override that skips even checking the options for certain tasks? Like a setting that can be overridden in SaaS but not the other environments? That way we can ship this to other envs without loading down SaaS.

Reply:
We could use a django setting that defaults to False, and that we can enable in S4S.

Reply:
That sounds good to me, added a setting for enabling/disabling ingest namespaces